PayButton/paybutton
# [#349] chore: update storybook

Related to #349





Description
---
Fixes #349 by updating storybook and addressing the migrations requirements posted at [the migration guide.](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-65x-to-700)


Test plan
---
Make sure the storybook runs normally. For that:
1. Go to the `react/` folder
2. Make sure you don't have cached containers (these names may differ in your machine):
  i. `docker container stop client-paybutton-react-1`
  ii. `docker container rm client-paybutton-react-1`
  iii. `docker image rm client-paybutton-react`
4. Run `docker-compose up` and access the storybook in the browser, should be working.
